pages/events/index.vue

```diff
@@ -292,6 +312,8 @@ function transformEsResponseToFilterGroups(aggregations: Aggregations): FilterGr
   return filterGroups
 }
 
+const filterDropdownSelectedFilters = ref({ 'ftvaEventTypeFilters.title.keyword': [], 'ftvaScreeningFormatFilters.title.keyword': [] })
```

Comment:
There is a ref already named userFilterSelection which is added to the page, can we replace this with using that instead. It is declared as `const userFilterSelection = ref<FilterItem>({})` If you load this  [URL](https://deploy-preview-76--test-ftva.netlify.app/events?dates=&filters=ftvaEventTypeFilters.title.keyword:(Guest+speaker+OR+35mm)&view=calendar) in new browser window you will see the selected filters are 0 in the dropdownfilters component

Reply:
got this done, just had to change a bit of logic to get it to work (was getting errors when I first tried it this way)